CONTRIBUTING.md

```diff
@@ -2,12 +2,12 @@
 
 ## Writing
 
--   One sentence per line.
-    No linewraps.
--   Inline FIRRTL code snippets should be tagged as FIRRTL code with `{.firrtl}`.
-    -   This isn't recognized by GitHub, but is used for the PDF generation.
--   Match terminology and capitalization preferences used elsewhere by default.
--   Don't forget to spell-check!
+- One sentence per line.
+  No linewraps.
+- Inline FIRRTL code snippets should be tagged as FIRRTL code with `{.firrtl}`.
+  - This isn’t recognized by GitHub, but is used for the PDF generation.
```

Comment:
I noticed a number of changes to non-ascii Unicode. Is this intended? Does this imply any change to the way we need to write the markdown?

Reply:
should be much better.